packages/frontend/editor-ui/src/features/settings/sso/provisioning/composables/userRoleProvisioning.store.ts

```diff
@@ -12,6 +12,14 @@ export const useUserRoleProvisioningStore = defineStore('userRoleProvisioning',
 
 	const provisioningConfig = ref<ProvisioningConfig | undefined>();
 
+	const isProjectRoleProvisioningEnabled = computed(
```

Comment:
I would suggest to have a dedicated `useUserRoleProvisionEnabled` composable inside `collaboration/projects`, since this one here contains the methods needed by the forms rendered in `SettingsSso.vue`.
That also makes the import tree leaner (for cached turbo tasks).

Looking at the code right now, I'd expect the component to never render, because no one is actually calling the `getProvisioningConfig` of this composable, which is what will save the returned value inside `provisioningConfig` of the composable.

Reply:
Only admins have the possibility to change this setting. So this is also not a piece of logic where I would worry about the state in the small composable for checking enabled and the settings composable diverging when the form is saved (until the next reload).